controller/array_action/array_mediator_ds8k.py

```diff
@@ -461,10 +463,13 @@ def unmap_volume(self, volume_id, host_name):
                 logger.debug("Successfully unmapped volume from host with lun {}.".format(lunid))
             else:
                 raise array_errors.ObjectNotFoundError(volume_id)
-        except exceptions.NotFound:
-            raise array_errors.HostNotFoundError(host_name)
+        except exceptions.NotFound as ex:
+            if HOST_DOES_NOT_EXIST in str(ex.message).upper():
+                raise array_errors.HostNotFoundError(host_name)
+            elif MAPPING_DOES_NOT_EXIST in str(ex.message).upper():
```

Comment:
same

Reply:
please include a link when referring to other conversations

Reply:
ok